deploy/iso/minikube-iso/package/Config.in

```diff
@@ -1,7 +1,5 @@
 menu "System tools"
-    source "$BR2_EXTERNAL_MINIKUBE_PATH/package/conmon/Config.in"
     source "$BR2_EXTERNAL_MINIKUBE_PATH/package/crio-bin/Config.in"
-    source "$BR2_EXTERNAL_MINIKUBE_PATH/package/sysdig/Config.in"
```

Comment:
A few packages could be removed because they were added to the newer buildroot (and otherwise cause redefinition errors)